<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>motores</class>
 <widget class="QDialog" name="motores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>331</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame
{
background-color: black;
color: white;
border-style: solid;
border-width:1px;
border-color: rgb(0, 167, 250);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Regulador</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="regulador_imagen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#regulador_imagen
{
background-image: url(:/gifs/1 Carcamo y Regulador/screen800x600/gifs/1 Carcamo y Regulador/carcamo hi quiet water.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox
{
background-color: black;
color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Manual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Automatico</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>351</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2
{
background-color: black;
color: white;
border-style: solid;
border-width:1px;
border-color: rgb(0, 167, 250);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Reactor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="reactor_imagen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#reactor_imagen
{
background-image: url(:/gifs/2 Reactor/screen800x600/gifs/2 Reactor/reac hi quiet water 03.png);
background-repeat: none;
background-position: center;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="gif_blower">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#gif_blower
{
background-image: url(:/gifs/0 Equipos/screen800x600/gifs/0 Equipos/Blower Jet off.png);
background-color: transparent;
background-repeat: none;
background-position:center;
border:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox_2
{
background-color: black;
color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Manual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Automatico</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QPushButton" name="asa_logo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>550</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">#asa_logo{ 
background-image: url(:/iconos/screen800x600/iconos/ASA logo.png);
border: none; 
background-repeat: none;
background-position: center;
} </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_3
{
background-color: black;
color: white;
border-style: solid;
border-width:1px;
border-color: rgb(0, 167, 250);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>4</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Alto general, presionar durante 3 segundos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton
{
background-color:transparent;
background-image: url(:/iconos/screen800x600/iconos/BOTON.png);
background-position: center;
background-repeat:none;
}
#pushButton::pressed
{
background-color:transparent;
background-image: url(:/iconos/screen800x600/iconos/BOTON 2.png);
background-position: center;
background-repeat:none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_4">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_4
{
background-color: black;
color: white;
border-style: solid;
border-width:1px;
border-color: rgb(0, 167, 250);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Filtro</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filtro_imagen">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>50</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#filtro_imagen
{
background-image: url(:/diagrama/screen800x600/diagrama/Filtro 5SDF portátil_ small.png);
background-repeat: none;
background-position: center;
backgorund-color: transparent;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox_3
{
background-color: black;
color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Manual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Automatico</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3"/>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>175</y>
     <width>331</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_5
{
background-color: black;
color: white;
border-style: solid;
border-width:1px;
border-color: rgb(0, 167, 250);
border-radius: 10px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white</string>
    </property>
    <property name="text">
     <string>Bomba de Alimentación</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="filtro_imagen_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>131</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#filtro_imagen_2
{
background-image: url(:/diagrama/screen800x600/diagrama/motor_filtro_small.png);
background-repeat: none;
background-position: center;
backgorund-color: transparent;
border: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>35</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#comboBox_4
{
background-color: black;
color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Manual</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Automatico</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
